expr/plan.go

```diff
@@ -198,15 +198,18 @@ func newplanFunc(e *expr, fn GraphiteFunc, context Context, stable bool, reqs []
 	// this function, we can set up the input arguments for the function
 	// that are series
 	pos = 0
-	for _, argExp = range argsExp[:cutoff] {
+	for _, argExp = range argsExp {
+		if pos >= len(e.args) {
+			break // no more args specified. we're done.
+		}
```

Comment:
the removal of `cutoff` here is this safe?

Reply:
looks good :) the argument consumption/iterating code is probably my least favorite code of MT. FWIW.
if you can handle this, you can handle anything else :dancing_men: